precalculate-measurements.go

```diff
@@ -0,0 +1,104 @@
+/*
+Copyright (c) Edgeless Systems GmbH
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	measuredboot "github.com/edgelesssys/uplosi/measured-boot"
+	"github.com/edgelesssys/uplosi/measured-boot/measure"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+)
+
+func newPrecalculateMeasurementsCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "precalculate-measurements <image>",
+		Short: "Precalculate TPM PCR measurements for an image",
+		Args:  cobra.ExactArgs(1),
+		RunE:  runPrecalculateMeasurements,
+	}
+	cmd.Flags().StringP("output-file", "o", "", "Output file for the precalculated measurements")
+	cmd.Flags().StringP("uki-path", "u", measuredboot.UkiPath, "Path to the UKI file in the image")
+
+	return cmd
+}
+
+func runPrecalculateMeasurements(cmd *cobra.Command, args []string) error {
+	flags, err := parsePrecalculateMeasurementsFlags(cmd)
+	if err != nil {
+		return fmt.Errorf("parsing flags: %w", err)
+	}
+
+	fs := afero.NewOsFs()
+	dissectToolchain := loadToolchain("DISSECT_TOOLCHAIN", "systemd-dissect")
+
+	simulator, err := measuredboot.PrecalculatePCRs(fs, dissectToolchain, flags.ukiPath, args[0])
+	if err != nil {
+		return fmt.Errorf("precalculating PCRs: %w", err)
+	}
+
+	if flags.outputFile != "" {
+		if err := writeOutput(fs, flags.outputFile, simulator); err != nil {
+			return fmt.Errorf("writing output: %w", err)
+		}
+		cmd.Printf("Wrote precalculated measurements to %s\n", flags.outputFile)
+	}
```

Comment:
If the flag is empty, we should print the measurements instead of just returning nil (unless that already happened somewhere else)